fdmi/source_dock_internal.h

```diff
@@ -105,6 +115,11 @@ struct m0_fdmi_src_dock {
 	 */
 	struct m0_tl          fsdc_posted_rec_list;
 
+	/**
```

Comment:
do you think it would be better to put this comment into a single line?

Reply:
OK